### DISCUSSION TITLE
Downloading Speed is really slow.

### DISCUSSION THREAD

**Discussion by bennyant** [aclap-dev/video-downloadhelper#946] [Help and Q&A]
### Your question

Hello, I have purchased the premium, but the download speed is still very slow (900 kb/s). It **used to be 5 mb/s**. Why has the speed gone down? Are there any settings that I have to change to increase the download speed limit?

**FYI,

### Addon & CoApp version and details

version: 9.0.2.12a1
target: mozilla
channel: beta
lang: en-GB
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"7936ff5965234eff"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "E:\\Files\\Kv",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "E:\\Files\\Kv"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 4,
    "successfull_dl": 25
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

If you want to talk about download speed, you have to provide some URLs on which you consistently get slow downloads.  I do not suffer from slow speeds so your claim that things have slowed down must be local to you.  So let's have some URLs & let's see if things are slow for me as well.

**Reply by GrampaWildWilly** [reply]
This might be as good a place as any to post this observation.

In discussion #950 I downloaded a 10-hour YouTube video of a bit over 9Meg in about 12 minutes.  Resource Monitor & VDH both showed I was getting about 14 million bytes per second download speed.  Just now I downloaded a 1-hour YouTube video of about 1.5Meg in about 8 minutes.  Resource Monitor & VDH both showed I was getting about 3.25 million bytes per second download speed.

This is still better than our situation about 2 years ago with VDH 7 when the time to download would always be about half whatever the duration of the YouTube video was.  Back in those days, my 1-hour video would have taken 30 minutes to download.  The 10-hour video would have taken 5 hours, which by the way would have discouraged me from even attempting to download it.

So even though the situation with YouTube videos is much improved compared to the past, it looks like the wankers at YouTube are still throttling us.  The speed they allow is some function of the duration of the video, but it's not entirely clear to me what the function is.  It's not as bad as it used to be, but it's still not as good as it should be.  This is entirely the fault of YouTube.  VDH will run at whatever speed YouTube will allow.  Other web sites give more consistent service.  I don't know why YouTube can't just give us proper service on everything.  They may think they are preserving their resources in order to give fair service to the greatest number of users.  But what saving are they getting by forcing me to tie up their resources for 8 minutes when they could give me much, much faster download speed & tie up resources for only a minute or so?  They are just not thinking clearly.
